### DISCUSSION TITLE
[Bug Report] @Bot play "D:\Somesong.mp3"   does not work as expected.

### DISCUSSION THREAD

**Discussion by stolencatkarma** [jagrosh/MusicBot#1569] [Troubleshooting]
### Bug Description

@Bot play "D:\Somesong.mp3" 

does not work as expected.

### Steps to Reproduce

have .mp3 file on your local hardrive. 
@ the bot with a local file
Bot plays some random youtube song.

### Expected Result

local file plays without needing to put it into a playlist

(playlists work as expected)

### Debug Output

Latest Adoptium on win10

### Checklist

- [X] I have looked for information about this [within the documentation](https://jmusicbot.com/)
- [X] I have searched for similar issues [on the issues page](https://github.com/jagrosh/MusicBot/issues?q=is%3Aissue)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by MichailiK** [comment]
Hi, playing local files with the `play` command seems to work fine on JMusicBot 0.4.1:

![IMG_5439](https://github.com/jagrosh/MusicBot/assets/39029839/db81ff8b-8e4e-4e3a-a10d-bd72d8b1bcd2)

Can you try doing `@bot play D:\something.mp3`(without the quotes, just the path?)

If that doesn’t work, we'll likely need more information, such as:
- Logs in the console
- Whether your bot is running in your Desktop PC or on a server
- Output of the `debug` command

**Comment by stolencatkarma** [comment]
works without the quotes. I guess I never tried it that way. makes sense though because there's no quotes in the Playlist files.

I guess I would only suggest to making the documentation on playing local files a bit clearer.
